src/h5web/visualizations/heatmap/HeatmapVis.module.css

```diff
@@ -30,6 +30,29 @@
   pointer-events: none;
 }
 
+.textureLoader {
+  background: rgba(0, 0, 0, 0.4)
+    linear-gradient(
+      to right,
+      transparent,
+      rgba(255, 255, 255, 0.5) 60%,
+      transparent
+    ) -50% top / 35% 100% no-repeat;
+  transition: opacity 0.2s ease-out;
+  opacity: 0;
+}
+
+.textureLoader[data-visible] {
```

Comment:
I use an attribute to show/hide the element instead of just removing it from the DOM so that it can fade out.